### ISSUE TITLE
Enhance VGG16 model

### ISSUE THREAD

**Issue by bottydim** [fchollet/deep-learning-models#89]
The original paper states that "The training was regularised by weight decay (the L2 penalty multiplier set to 5 · 10−4) and **dropout regularisation** for the first two fully-connected layers (dropout ratio set to 0.5).

I think we have forgotten to add them:

        x = Dense(4096, activation='relu', name='fc1')(x)
        x = Dense(4096, activation='relu', name='fc2')(x)
        x = Dense(classes, activation='softmax', name='predictions')(x)
